rasa_utils/bot_server_channel.py

```diff
@@ -150,10 +150,10 @@ def tracker(cid):
 
         @custom_webhook.route("/conversations/<cid>/say", methods=["GET"])
         def say(cid):
-            message = request.args.get(b"message", [])
-            _payload = request.args.get(b"payload", [])
-            _display_name = request.args.get(b"display_name", [])
-            _uuid = request.args.get(b"uuid", [])
+            message = bytes(request.args.get("message"), "utf8")
```

Comment:
The `message` object is missing a default value for when it is empty. Similarly, to the `payload` etc below.

Reply:
Yes, the old Klein server used parsed its arguments as lists. Should be strings now.